sql/connect/client/jvm/pom.xml

```diff
@@ -175,21 +175,27 @@
                 <include>com.google.common.**</include>
               </includes>
             </relocation>
+            <relocation>
```

Comment:
Here, I've made the relocation rules for pb consistent with those on the server side, otherwise errors would occur during compilation and testing. Let me think about if there's a better way to handle this.